install/Application/src/Entity/User/AdminUser.php

```diff
@@ -8,10 +8,6 @@
 use Sylius\Component\Core\Model\AdminUser as BaseAdminUser;
 use Synolia\SyliusAdminOauthPlugin\Entity\User\CustomAdminUserTrait;
 
-/**
- * @ORM\Entity
- * @ORM\Table(name="sylius_admin_user")
- */
```

Comment:
I think we need to keep it for Sylius 1.12 because of https://github.com/synolia/SyliusAdminOAuthPlugin/actions/runs/12549387989/job/35585842377?pr=23#step:12:930 , no ?

Reply:
yes probably, i'll give it a try

Reply:
:100: